x-pack/metricbeat/module/sql/query/query.go

```diff
@@ -44,24 +51,36 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	cfgwarn.Beta("The sql query metricset is beta.")
 
 	config := struct {
-		Driver string `config:"driver" validate:"nonzero,required"`
-		Query  string `config:"sql_query" validate:"nonzero,required"`
+		Driver         string `config:"driver" validate:"nonzero,required"`
+		Query          string `config:"sql_query" validate:"nonzero,required"`
+		ResponseFormat string `config:"response_format"`
 	}{}
```

Comment:
You can predefine default values here, and then you don't need to set the value later if empty.
```suggestion
	}{
            ResponseFormat: tableResponseFormat,
	}
```

Reply:
Yeah I know but after the parsing the value with `base.Module().UnpackConfig(&config)` it becomes empty `""`, unless I'm doing something wrong

Reply:
Please forget what I said before :sweat_smile: